bench/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "sanctuary-type-classes-benchmarks",
+  "version": "0.0.0",
+  "private": true,
+  "license": "WTFPL",
```

Comment:
Perhaps it worked in this case because the versions differ: `5.2.0` versus `6.0.0`. If you upgrade the dependency you may see the error return.